docs/cugraph/source/index.rst

```diff
@@ -1,58 +1,84 @@
-   :widths: 25 25 25
-   :header-rows: 1
-
-   * - Core
-     - GNN
-     - Extension
-   * - :abbr:`cugraph (Python wrapper with lots of convenience functions)`
-     - :abbr:`cugraph-ops (GNN aggregators and operators)`
-     - :abbr:`cugraph-service (Graph-as-a-service provides both Client and Server packages)`
-   * - :abbr:`pylibcugraph (light-weight Python wrapper with no guard rails)`
-     - :abbr:`cugraph-dgl (Accelerated extensions for use with the DGL framework)`
-     -
-   * - :abbr:`libcugraph (C++ API)`
-     - :abbr:`cugraph-pyg (Accelerated extensions for use with the PyG framework)`
-     -
-   * - :abbr:`libcugraph_etl (C++ renumbering function for strings)`
-     - :abbr:`wholegraph (Shared memory-based GPU-accelerated GNN training)`
-     -
-..
-|
 
 ~~~~~~~~~~~~
 Introduction
 ~~~~~~~~~~~~
 cuGraph is a library of graph algorithms that seamlessly integrates into the
 RAPIDS data science ecosystem and allows the data scientist to easily call
 graph algorithms using data stored in GPU DataFrames, NetworkX Graphs, or
-even CuPy or SciPy sparse Matrices.
+even CuPy or SciPy sparse Matrices. Our major integration effort with NetworkX allows
+**zero code change** use of nx-cuGraph as a backend for NetworkX calls. This offers a near seamless
+transition to GPU accelerated graph analytics to NetworkX users with access to a supported GPU.
+
+Getting started with cuGraph
+
+Required hardware/software for cuGraph and `RAPIDS <https://docs.rapids.ai/user-guide>`_
+ * NVIDIA GPU, Volta architecture or later, with `compute capability <https://developer.nvidia.com/cuda-gpus> 7.0+`_
+ * CUDA 11.2-11.8, 12.0-12.5
+ * Python version 3.10, 3.11, or 3.12
+ * NetworkX version 3.0 or newer in order to use use the nx-cuGraph backend. Version 3.3 is required to use `NetworkX Configs <https://networkx.org/documentation/stable/reference/backends.html#module-networkx.utils.configs>`_ `see below <#cugraph-using-networkx-code>`_.
+
+Installation
+The latest RAPIDS System Requirements documentation is located `here <https://docs.rapids.ai/install#system-req>`_.
+
+This includes several ways to set up cuGraph
+
+* From Unix
+
+  * `Conda <https://docs.rapids.ai/install/#conda>`_
+  * `Docker <https://docs.rapids.ai/install/#docker>`_
+  * `pip <https://docs.rapids.ai/install/#pip>`_
+
+
+**Note: Windows use of RAPIDS depends on prior installation of** `WSL2 <https://learn.microsoft.com/en-us/windows/wsl/install>`_.
+
+* From Windows
+
+  * `Conda <https://docs.rapids.ai/install#wsl-conda>`_
+  * `Docker <https://docs.rapids.ai/install#wsl-docker>`_
+  * `pip <https://docs.rapids.ai/install#wsl-pip>`_
 
-Note: We are redoing all of our documents, please be patient as we update
-the docs and links
 
-|
+Build From Source
+
+To build from source, check each RAPIDS GitHub README for set up and build instructions. Further links are provided in the `selector tool <https://docs.rapids.ai/install#selector>`_.
```

Comment:
> check each RAPIDS GitHub README for set up and build instructions

Is this referring to each of the separate Graph projects' repos in preparation for the migration?

Also, I don't think the install selector tool is related to building from source.